priv/scripts/otp/otp.sh

```diff
@@ -39,12 +39,14 @@ docker cp ${container}:/home/build/out/${ref_name}.tar.gz ${ref_name}.tar.gz
 
 docker rm -f ${container}
 
+build_sha256=$(sha256sum ${ref_name}.tar.gz | cut -d ' ' -f 1)
+
 aws s3 cp ${ref_name}.tar.gz s3://s3.hex.pm/builds/otp/${arch}/${linux}/${ref_name}.tar.gz --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"builds builds/otp builds/otp/${arch} builds/otp/${arch}/${linux} builds/otp/${arch}/${linux}/${ref_name}\",\"surrogate-control\":\"public,max-age=604800\"}"
 
 aws s3 cp s3://s3.hex.pm/builds/otp/${arch}/${linux}/builds.txt builds.txt || true
 touch builds.txt
 sed -i "/^${ref_name} /d" builds.txt
-echo -e "${ref_name} ${ref} $(date -u '+%Y-%m-%dT%H:%M:%SZ')\n$(cat builds.txt)" > builds.txt
+echo -e "${ref_name} ${ref} $(date -u '+%Y-%m-%dT%H:%M:%SZ') ${build_sha256}\n$(cat builds.txt)" > builds.txt
```

Comment:
since we didn't have this field yet, I'd use this opportunity to make it super explicit what it is:

```suggestion
echo -e "${ref_name} ${ref} $(date -u '+%Y-%m-%dT%H:%M:%SZ') sha256:${build_sha256}\n$(cat builds.txt)" > builds.txt
```

That being said, elixir builds.txt does NOT have this `sha256:` prefix so perhaps it's better to keep it consistent.

Reply:
I agree, the tag would be nice.  However, I think consistency would be more important at this point.